src/playground/copyStaticAssets.ts

```diff
@@ -0,0 +1,6 @@
+import * as shell from 'shelljs';
+
+shell.mkdir('-p', 'dist/public/');
+shell.cp('-R', 'src/public/assets/css', 'dist/public/');
+shell.cp('-R', 'src/public/assets/js', 'dist/public/');
+shell.cp('-R', 'src/views', 'dist/');
```

Comment:
We've been using the native Node `fs` library for this sort of thing - https://github.com/xFAANG/askql/blob/master/scripts/postBuild.ts. Do you think it's worth it to add this additional dependency?

Reply:
Yes, becase surprisingly, there is no function to copy the whole directory in `fs`.
https://nodejs.org/api/fs.html

Please use their own functions (which may contain bugs) or one of libraries.
E.g. these: https://stackoverflow.com/questions/13786160/copy-folder-recursively-in-node-js

Reply:
Ok, cool. However, I think this should be a part of https://github.com/xFAANG/askql/tree/master/scripts

Reply:
Done